### ISSUE TITLE
Existing pyspark installation not detected during pip install delta-spark (or pip install pyspark)

### ISSUE THREAD

**Issue by willim32533** [bitnami/bitnami-docker-spark#65]
Using the following Dockerfile to extend the latest bitnami spark image, results in pyspark being installed, which should not happen, since it is already installed.  Is there a particular python environment variable missing, or is there a specific way needed for installing python packages?

```
FROM bitnami/spark:latest
USER root
RUN pip install delta-spark
```

**Comment by willim32533** [comment]
FWIW - pip install pyspark will also install pyspark, even on the latest bitnami spark image.

**Comment by willim32533** [comment]
Thank you, I sincerely appreciate the response and workaround!

**Comment by migruiz4** [comment]
Hi @willim32533,

I'm happy you found my suggestion helpful.

We have added the environment variable `PYTHONPATH` to the base image, so the workaround shouldn't be needed anymore. 

The new release including the changes should be released soon (`bitnami/spark:3.2.0-debian-10-r80`).
